libs/markets/src/lib/components/market-info/market-info-panels.tsx

```diff
@@ -191,6 +209,115 @@ export const KeyDetailsInfoPanel = ({ market }: MarketInfoProps) => {
   );
 };
 
+const SuccessionLineItem = ({
+  marketId,
+  isCurrent,
+}: {
+  marketId: string;
+  isCurrent?: boolean;
+}) => {
+  const { data, loading } = useSuccessorMarketQuery({
+    variables: {
+      marketId,
+    },
+  });
+
+  const marketData = data?.market;
+  const governanceLink = useLinks(DApp.Token);
+  const proposalLink = marketData?.proposal?.id
+    ? governanceLink(TOKEN_PROPOSAL.replace(':id', marketData?.proposal?.id))
+    : undefined;
+
+  if (loading) {
+    return (
+      <div className="rounded p-2 bg-vega-clight-700 dark:bg-vega-cdark-700 h-[68px] animate-pulse"></div>
+    );
+  }
+  return (
+    marketData && (
```

Comment:
**thought (non-blocking)**: Based on a previous testnet run, it's possible that we have successor market IDs but no longer have any information about the markets. I think it would be worth showing the market IDs anyway rather than showing nothing at all.